german/schwimmen/hunger/bierkeller/bierkeller.md

```diff
@@ -0,0 +1 @@
+Sie schwimmen zum schoenen fraenkischen Bierkeller am Rande des Ufers. Dort bestellen Sie sich erstmal "a Seidla" und "a Kellerbladdn". Gestaerkt ueberlegen Sie sich was als naechstes zu tun ist.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.